fixtures/appliance.py

```diff
@@ -0,0 +1,82 @@
+""" This module contains fixtures to use when you need a temporary appliance for testing.
+
+In cases where you cannot run a certain test againts the primary appliance because of the test's
+destructive potential (which could render all subsequent testing useless), you want to use
+a temporary appliance parallel to the primary one.
+
+For tests such as setting up an appliance (or multiple) in a certain setup, you will want to use
+the :py:func:`temp_appliance_unconfig` fixture and then configure the appliance(s) yourself.
+
+For tests where all you need is a single preconfigured appliance to run a database restore on, for
+example, you will want to use the :py:func:`temp_appliance_preconfig` fixture.
+"""
+from contextlib import contextmanager
+
+import pytest
+
+from cfme.test_framework.sprout.client import SproutClient
+
+
+@contextmanager
+def _temp_appliance(preconfigured=True, lease_time=180):
+    """ Provisions a temporary appliance for testing
+
+    Args:
+        preconfigured: True if the appliance should be already configured, False otherwise
+        lease_time: Lease time in minutes (3 hours by default)
+    """
+    sprout_client = SproutClient.from_config()
+    apps, request_id = sprout_client.provision_appliances(
+        lease_time=lease_time, preconfigured=preconfigured)
+    app = apps[0]
+    app.stop_evm_service()
+    app.extend_db_partition()
+    app.start_evm_service()
+    yield app
+    sprout_client.destroy_pool(request_id)
```

Comment:
**OPTIONAL**: `try/finally` so destroy_pool is in finally? (I hope it works like that in CMs ... ). Can be done in another PR, this is just a suggestion